www/shia02/HW7/js/main.js

```diff
@@ -0,0 +1,126 @@
+var gameField = document.querySelector('#game-field');
+var points = document.querySelector('#points');
+
+var numberOfPoints = 0;
+var foundCitiesCount = 0;
+
+var firstClickedCard = null;
+var secondClickedCard = null;
+
+var cities = ['Prague', 'Moscow', 'Brussels', 'Dublin', 'London', 'Havana', 'Monaco', 'Oslo', 'Paris', 'Rome'];
+cities = cities.concat(cities);
+cities.sort(function () {
+    return 0.5 - Math.random();
+});
+
+var incrementPoints = function() {
+    numberOfPoints++;
+
+    points.innerText = numberOfPoints;
+};
+
+var decrementPoints = function() {
+    if (numberOfPoints >= 1) {
+        numberOfPoints--;
+    }
+
+    points.innerText = numberOfPoints;
+};
+
+var createCard = function (name) {
+    var card = document.createElement('div');
+    card.classList.add('card');
+
+    var span = document.createElement('span');
+    span.innerText = name;
+    span.hidden = true;
```

Comment:
jsem ani nevedel ze vlastnost `hidden` existuje, mozna ale by bylo lepsi to resit pres CSS display: none
```css
.card span {
   display: none;
}
.card.revealed span {
   display: block;
}
```